.github/workflows/stable.yml

```diff
@@ -101,39 +101,36 @@ jobs:
 
   quiche_ios:
     runs-on: macos-latest
+    strategy:
+      matrix:
+        target: ["x86_64-apple-ios", "aarch64-apple-ios"]
     steps:
       - name: Checkout sources
         uses: actions/checkout@v2
         with:
           submodules: 'recursive'
 
-      - name: Install stable toolchain x86_64
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: ${{ env.TOOLCHAIN }}
-          target: x86_64-apple-ios
-          override: true
-
-      - name: Install stable toolchain aarch64
+      - name: Install stable toolchain
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: ${{ env.TOOLCHAIN }}
-          target: aarch64-apple-ios
+          target: ${{ matrix.target }}
           override: true
 
       - name: Install cargo-lipo
-        run: cargo install cargo-lipo
+        uses: actions-rs/install@v0.1
+        with:
+          crate: cargo-lipo
 
       - name: Run cargo lipo
-        uses: actions-rs/cargo@v1
-        with:
-          command: lipo
-          args: --verbose
+        run: cargo lipo --verbose --targets ${{ matrix.target }}
```

Comment:
Isn't the whole point of lipo that it builds a universal binary with all the architectures? Otherwise if we are just building each architecture one by one we could just use `cargo build`, no?

Reply:
Right. We can go back to the previous state but I removed cargo-lipo now. It doesn't build a universal binary any more, but probably enough for testing a build.